fdbclient/BackupAgentBase.actor.cpp

```diff
@@ -695,6 +700,39 @@ Future<Void> readCommitted(Database cx,
 	    cx, results, Void(), lock, range, groupBy, Terminator::True, AccessSystemKeys::True, LockAware::True);
 }
 
+ACTOR Future<Void> sendCommitTransactionRequest(CommitTransactionRequest req,
+                                                Key uid,
+                                                Version newBeginVersion,
+                                                Key rangeBegin,
+                                                NotifiedVersion* committedVersion,
+                                                int* totalBytes,
+                                                int* mutationSize,
+                                                PromiseStream<Future<Void>> addActor,
+                                                FlowLock* commitLock,
+                                                PublicRequestStream<CommitTransactionRequest> commit) {
+	Key applyBegin = uid.withPrefix(applyMutationsBeginRange.begin);
+	Key versionKey = BinaryWriter::toValue(newBeginVersion, Unversioned());
+	Key rangeEnd = getApplyKey(newBeginVersion, uid);
+
+	req.transaction.mutations.push_back_deep(req.arena, MutationRef(MutationRef::SetValue, applyBegin, versionKey));
```

Comment:
In case this is not already the case, it should be documented where mutations and encryptedMutations are defined that their contents must always be paired by index.

Reply:
I had already documented these in `CommitTransaction.h` so add a comment pointing there